domains/primitives/messenger/src/lib.rs

```diff
@@ -27,10 +27,16 @@ use crate::messages::MessageKey;
 #[cfg(not(feature = "std"))]
 use alloc::vec::Vec;
 use codec::{Decode, Encode};
+use frame_support::__private::sp_inherents;
+use frame_support::__private::sp_inherents::Error;
```

Comment:
Please use the `sp-inherents` crate instead of something with explicit `__private`.

Reply:
my IDE is the culprit. Good catch!